src/App.jsx

```diff
@@ -14,70 +16,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function toogleSelect(selectGood) {
```

Comment:
name of param `selectGood` is verb
a parameter must be a noun
you can rename it to `selectedGood`, but it better just `good`

Reply:
it's better when the function does one action
in your case the function sets and clears the state
there is a function to establish, so this function repeats the logic
It’s better to move the cleanup logic into a separate function